### PR TITLE
ZEN-30423 - white/dartk themes support for reports.

### PR DESCRIPTION
**Pull request by bmoskalyk**

### CODE REVIEW

File: Products/ZenUI3/browser/resources/js/zenoss/Reports.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ report_panel = Ext.create('Ext.panel.Panel', {
     items: [{
         xtype: 'backcompat',
         ref: 'backcompat',
-        id: 'report_backcompat'
+        id: 'report_backcompat',
+        iframeCustomCss: 'z-cse-reports'
```
Comment (bmoskalyk) [inline Products/ZenUI3/browser/resources/js/zenoss/Reports.js]: inject custom css to iframe html to add css styles to reports.

File: Products/ZenUI3/browser/resources/css/zen-cse.css [hunk 2]
```diff
@@ -2943,3 +2990,44 @@ BYE BYE BORDERS
     .z-cse.z-cse-dark .edited_feedback .x-panel-header-text {
         color: yellow !important;
     }
```
Comment (carlcamera) [inline Products/ZenUI3/browser/resources/css/zen-cse.css]: 'yellow' is a bit too harsh.  Match the new flare background color #cdf856 instead.

File: Products/ZenUI3/browser/resources/css/zen-cse.css [hunk 3]
```diff
@@ -2255,6 +2255,53 @@ BYE BYE BORDERS
       visibility: hidden;
     }
 
+    /*<ZEN-30423> white theme styles for reports start*/
+    .z-cse.z-cse-reports th.tabletitle div.tabletitlecontainer {
+        background-color: #d2d2d2 !important;
```
Comment (carlcamera) [inline Products/ZenUI3/browser/resources/css/zen-cse.css]: I would prefer to have transparent background colors.  Then the background of the application behind the report will be shown.  We should discuss.

File: Products/ZenUI3/browser/resources/css/zen-cse.css [hunk 4]
```diff
@@ -2255,6 +2255,53 @@ BYE BYE BORDERS
...
+        vertical-align: middle;
+    }
+    .z-cse.z-cse-reports a.tableheaderselected,
+    .z-cse.z-cse-reports a.tableheader {
+        display: block;
+    }
+    .z-cse.z-cse-reports th.tableheaderselected,
+    .z-cse.z-cse-reports th.tableheader {
+        background-image: url(/++resource++zenui/img/xtheme-zenoss/grid/grid3-hrow.png) !important;
+        background-color: #ccc !important;
```
Comment (carlcamera) [inline Products/ZenUI3/browser/resources/css/zen-cse.css]: I am wary of any files from the old xtheme-zenoss days.  There are no beveled backgrounds in the new UI.  Flat. Flat. Flat.

File: Products/ZenUI3/browser/resources/css/zen-cse.css (hunk 3 above)
Comment (bmoskalyk) [reply Products/ZenUI3/browser/resources/css/zen-cse.css]: you are right transparent background is that what we need there.

File: Products/ZenUI3/browser/resources/css/zen-cse.css (hunk 2 above)
Comment (bmoskalyk) [reply Products/ZenUI3/browser/resources/css/zen-cse.css]: changed.

File: Products/ZenUI3/browser/resources/css/zen-cse.css (hunk 4 above)
Comment (bmoskalyk) [reply Products/ZenUI3/browser/resources/css/zen-cse.css]: changed.

**Pull request merged**